src/strings.json

```diff
@@ -85,5 +85,9 @@
   "collapsePanes": "Collapse panes",
   "expandPanes": "Expand panes",
   "copySourceUrl": "Copy Source Url",
-  "copySourceUrl.accesskey": "c"
+  "copySourceUrl.accesskey": "c",
+  "shift": "\u21E7",
+  "cmd": "\u2318",
+  "alt": "\u2325",
```

Comment:
Hmm now that i'm looking more closely perhaps we don't need to localize the unicode keys :) shift, cmd, alt...